Makefile

```diff
@@ -431,10 +446,20 @@ ifndef VERSION
 endif
 	git tag $(VERSION)
 
-	CALICOCTL_NODE_VERSION=$(VERSION) $(MAKE) dist/calicoctl calico/ctl calico/node
+	# Build the calicoctl binaries, as well as the calico/ctl and calico/node images.
+	CALICOCTL_NODE_VERSION=$(VERSION) $(MAKE) dist/calicoctl dist/calicoctl-darwin-amd64 dist/calicoctl-windows-amd64 
+	CALICOCTL_NODE_VERSION=$(VERSION) $(MAKE) calico/ctl calico/node
+
 	# Check that the version output includes the version specified.
-# Tests that the "git tag" makes it into the binary. Main point is to catch "-dirty" builds
-	if ! dist/calicoctl version | grep -P 'Version:\s*$(VERSION)$$'; then echo "Reported version:" `dist/calicoctl version` "\nExpected version: $(VERSION)"; false; else echo "Version check passed\n"; fi
+        # Tests that the "git tag" makes it into the binaries. Main point is to catch "-dirty" builds
```

Comment:
Indentation is off.

It was the way it was before so that some lines were echoed and some weren't.